apps/client/src/common/components/schedule/Schedule.tsx

```diff
@@ -34,8 +34,8 @@ export default function Schedule({ className }: ScheduleProps) {
           <ScheduleItem
             key={event.id}
             selected={selectedState}
-            timeStart={event.timeStart}
-            timeEnd={event.timeEnd}
+            timeStart={event.timeStart + (event?.delay || 0)}
+            timeEnd={event.timeEnd + (event?.delay || 0)}
```

Comment:
Not such a big deal, but `||` will default to the `0` even if `event?.delay` is `0` itself. If you want to avoid that, use `??`

Reply:
great suggestion